### ISSUE TITLE
Does this really work anymore

### ISSUE THREAD

**Issue by jonaswikstrom** [MSkjel/LocalPulse2Tibber#7]
Hello

I did configured a Tibber Pulse with success a couple of month ago. Now I have another Pulse trying to reproduce the steps but at action number 19 it is stop. The app does not seem to accept the Pulse as a configured unit when starting the app again. I have tried serveral times without success. 

Any information?

**Comment by MSkjel** [comment]
Hmm, I dont see any obvious changes to their app. Have you tried with an older version of it? I did all this using v22.42.0. You are also sure you are hitting Send before hitting Apply on the pulse website? You could also try and wait a bit after doing everythinng and see if it just pops up in the app after a while.

**Comment by ProfDrYoMan** [comment]
@jonaswikstrom you might also check out this one which is getting rid of a lot of steps in between and works flawless for me.

https://github.com/marq24/ha-tibber-pulse-local

**Comment by jswetzen** [comment]
I just set it up successfully with Tibber Pulse P1 and version 23.42.0 of the Android app. Got local MQTT and Tibber forwarding working beautifully as far as I can see (ten minutes in...) Thanks @MSkjel !

**Comment by Hant0r** [comment]
It's the same for me as it is for @jonaswikstrom 
As soon as I restart the Pulse after changing the data, it falls into an error and no longer starts.

Unfortunately, @ProfDrYoMan solution doesn't help me because I don't have HA. Are there any other solutions without HA, e.g. with ioBroker?

**Comment by MrStrategy** [comment]
same here. 
But it looks like the tibber bridge needs a tls connection. You can't set the certificate information in the params to NULL any more and the dmesg log within the bridge indicates, it still tries to use TLS authentication. 
I'll add some more details once I'm able to access the device again.

**Comment by G4mm3l** [comment]
I've run into the same issues, i'am trying to set generated self signed client certificates over the web ui, but it still says that there's an issue in formatting (error: SyntaxError: Bad control character in string literal in JSON at position 3146 (line 1 column 3147))
Has anyone solved this problem?

**Comment by ingoblum** [comment]
There seems to be a parsing issue with the current web ui of the bridge, however it is also possible to use the console on the third tab. With `help` you can show a list of available commands, `param_list` prints all the variables and their contents and finally
```
param_set ca_cert ''
param_set certificate ''
param_set private_key ''
```
cleared the certificates. After that the bridge connected to my Mosquitto instance successfully without any authentication. Don't forget to save to flash with either the button "Store params to flash" or the command `param_store`.

**Comment by hansahell** [comment]
I'm not able to finish the very last step under Pulse Setup in the end of this guide, the Tibber Pulse doesn't seem to want apply the latest changes and I guess that's maybe because the Tibber Pulse requires TLS connection? - ca_cert, certificate and private_key files are left empty. Does anyone know how to solve this?
![image](https://github.com/user-attachments/assets/b68fd143-2dc8-4bd4-bbdd-c548491dd446)

**Comment by jswetzen** [comment]
There's a step to extract the certficate, but maybe this was only for man-in-the-middle setup... Have you tried setting up a self-signed cert for MQTT and enter those there?

Johan Swetzén

-------- Ursprungligt meddelande --------
Den 2025-01-04 12:05, hansahell  skrev:

**Comment by hansahell** [comment]
Extracting the certificate is for the MQTT broker and I've done that.. This problem occurs at the very last step.
_>>Have you tried setting up a self-signed cert for MQTT and enter those there?_
No, I'm trying and hoping to avoid that but if I'm not able to find another solution, I will do that and test..

**Comment by MSkjel** [comment]
You don't need a certificate for it to work. Of no certificate is given the Pulse won't use TLS. You could of course try with a self-signed one if you want, but it isn't needed. You are sure the SSID and Password are correct, and you are using a 2.4GHz network?

**Comment by hansahell** [comment]
I've created an own dedicated 2.4GHz network for IoT devices and the password is correct. I've set this up before and it was working for a long time but then the Pulse had 1.2.3 version. Now when I tried to set this up again today, the Pulse was updated to 1.2.5 and the last step when adding the mqtt url, topic and sub_topic and clicking Send and Apply, it won't save for some reason... I'm wondering if it has something with the 1.2.5 version.

**Comment by MSkjel** [comment]
That might be, but I see the update url field is also removed. You could just try with the certificate from tibber, just to see if it saves the settings then?

**Comment by hansahell** [comment]
Yes, I will do some more testing tonight when the kids are in bed.

**Comment by hansahell** [comment]
I can now confirm that this guide still works fine (even with 1.2.5 FW version) :)
Step 19 took some time, meaning that it took 5-10 min before I saw the Pulse pop up in the app.
The last step also works fine, no certificates needed - I found my problem and it was a typo in my mqtt user password :-P

Thanks to both @jswetzen and @MSkjel for trying to help me :)

**Comment by ostwilkens** [comment]
Just chiming in to say it took 22 minutes before the Pulse turned up in the app for me (step 19)
